packages/apollo-cli/tsconfig.json

```diff
@@ -19,5 +19,5 @@
     "noUnusedLocals": false
   },
   "include": ["./src/**/*"],
-  "exclude": [ "**/__tests__/*" ]
+  "exclude": [ "**/__tests__/*", "**/__mocks__/*" ]
```

Comment:
Jest was complaining about having a JS and TS file for the mocks, so I just disabled compiling the mock files to JS.